cypress/e2e/helpers/surveyHelper.ts

```diff
@@ -82,6 +91,117 @@ export const verifyWeightScreenContent = (
   );
 };
 
+export const verifyOnboardingExperienceOptions = (
+  progressBarValue: string,
+  percentValue: string,
+  surveyName: string,
+) => {
+  cy.contains(BUTTON, CLOSE).should("be.visible");
+  cy.contains("h4", surveyName).should("be.visible");
+
+  const currentDate = new Date();
+  const formattedDateText = format(currentDate, "EEEE, MMMM dd");
+  cy.get(DATE_TEXT).should("contain", formattedDateText);
+
+  cy.contains(HOW_HAS_YOUR_ONBOARDING_EXPERIENCE_BEEN_SO_FAR);
+  cy.contains(REQUIRED);
+
+  cy.get(PROGRESS_BAR_VALUE_ID(progressBarValue)).should("exist");
+  cy.get(VALUE_PROGRESS_BAR_TEXT).should(
+    "contain",
+    PERCENT_NUMBER(percentValue),
+  );
+
+  cy.get(OPTION_RADIO_EXCELLENT).should("exist").should("not.be.checked");
+  cy.get(OPTION_RADIO_GOOD).should("exist").should("not.be.checked");
+  cy.get(OPTION_RADIO_CONFUSING).should("exist").should("not.be.checked");
+  cy.contains("button", NEXT).should("be.visible");
+};
+
+export const verifyPleaseProvideFeedbackScreen = (
```

Comment:
Also, just suggestions =)
```suggestion
export const verifyProvideFeedbackScreen = (
```

or 

```suggestion
export const verifyFeedbackScreen = (
```